src/commands/register.js

```diff
@@ -158,9 +160,12 @@ class RegisterCommand extends Command{
                     const otherMember = await interaction.guild.members.fetch(otherMemberDoc.user).catch(() => null);
                     if(otherMember) await otherMember.roles.add(role);
                 }
+                needUpdate = true;
             }
             await member.roles.add(role);
         }
+
+        if (needUpdate) client.emit('updateGuilds')
```

Comment:
explicacao no primeiro comentario em src/events/updateGuilds.js
```suggestion
        if (needUpdate) client.emit('updateGuilds', false);
```